entrypoint.sh

```diff
@@ -15,11 +15,11 @@ SM_COLOR="9CFF00"
 MD_COLOR="EFFF00"
 LG_COLOR="FF7500"
 XL_COLOR="C60404"
-LABEL_XS="size-xs"
-LABEL_SM="size-sm"
-LABEL_MD="size-md"
-LABEL_LG="size-lg"
-LABEL_XL="size-xl"
+LABEL_XS="${custom_tag_xs}"
```

Comment:
Not sure if this work or it should be like the other arguments, something like
```suggestion
LABEL_XS="$6"
```